setup.py

```diff
@@ -3,6 +3,13 @@
 from setuptools import setup, find_packages
 import subprocess
 
+with open("/etc/redhat-release") as fp:
```

Comment:
Can this be imported from major_version.py?

Reply:
Yeah I should be able to do that.